spec/requests/admin_claims_spec.rb

```diff
@@ -107,19 +107,31 @@
   end
 
   describe "claims#search" do
-    let(:claim) { create(:claim, :submitted) }
+    let(:claim1) { create(:claim, :submitted, surname: "Wayne") }
+    let(:claim2) { create(:claim, :submitted, surname: "Wayne") }
 
     it "redirects to a claim when one exists" do
```

Comment:
The code isn't redirecting so these words are no longer quite right. Also this spec appears to replicate the newly added spec below almost identically. Do we actually need both?

Reply:
I've tweaked this now, so we do redirect, so both specs are now legit